src/net/sourceforge/kolmafia/request/FightRequest.java

```diff
@@ -2323,6 +2323,17 @@ public static final void updateCombatData(
 
       KoLCharacter.getFamiliar().recognizeCombatUse();
 
+      FamiliarData familiar = KoLCharacter.getEffectiveFamiliar();
+      int familiarId = familiar.getId();
+      // If other familiars also end up getting charged at start of fight rather than end we can put
+      // them here
+      switch (familiarId) {
+        case FamiliarPool.GHOST_COMMERCE:
+          {
+            Preferences.increment("commerceGhostCombats");
+          }
```

Comment:
I'd like a break here ideally, just to avoid future issues

Reply:
Done.  Good catch.  Thank you.